### DISCUSSION TITLE
Welcome to WIFItally Discussions!

### DISCUSSION THREAD

**Discussion by DynamoBen** [DynamoBen/WIFItally#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TheBoojah** [comment]
![image](https://user-images.githubusercontent.com/33135012/113422672-68c12800-93cd-11eb-998f-83cbd9cc103b.png)
 I made one using a Wemos D1 mini, just soldered some leds directly to it and made a custom 3d printed case.
The board supplies more current (20mA) when sinking, so had to invert the code to light them properly.

Thanks for the software @DynamoBen! Used with Vmix TCP it just worked flawlessly.

**Reply by DynamoBen** [reply]
Looks great!
